atm.js

```diff
@@ -1,21 +1,63 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var checkingBalance_div = document.querySelector("div#checking_balance");
+var savingsBalance_div = document.querySelector("div#savings_balance");
+checkingBalance = 0;
+savingsBalance = 0;
+console.log(checkingBalance);
 
 var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+  checkingBalance += parseInt(amount);
+  checkingBalance_div.innerHTML = "$" + checkingBalance;
+  if(checkingBalance >= 0){
+    console.log("hi");
+    $("div#checking_balance").removeClass("zero");
+  }
+
```

Comment:
I love the use of console.log() throughout your program. One thing I would comment on, however, is to provide semantics for your logs. They should be stating what is happening when that part of the code is executing. As it stands in this current app what you did is fine. But if you were to debug a bigger problem on a grander scale you'll be using way more loggings to figure something out. That's where semantic logs will really help out